JSDemos/Demos/DateBox/Formatting/description.md

```diff
@@ -1 +1,17 @@
-This demo shows how to&nbsp;use a&nbsp;locale-dependent format (or&nbsp;default format), <a href="/Documentation/ApiReference/Common/Object_Structures/format/#type">built-in format</a>, <a href="/Documentation/Guide/Common/Value_Formatting/#Format_Widget_Values/Custom_Format_String">LDML&nbsp;pattern</a>, and a&nbsp;format with literal characters to&nbsp;specify the <a href="/Documentation/ApiReference/UI_Widgets/dxDateBox/Configuration/#displayFormat">displayFormat</a>. Note that when the <a href="/Documentation/ApiReference/UI_Widgets/dxDateBox/Configuration/#useMaskBehavior">useMaskBehavior</a> property is&nbsp;set to&nbsp;<b>true</b>, the <b>DateBox</b> formats the display value and also controls user input.
\ No newline at end of file
+To customize how a **DateBox** displays dates, use the [displayFormat](/Documentation/ApiReference/UI_Widgets/dxDateBox/Configuration/#displayFormat) property. This demo illustrates the following use cases:
+
+- **Locale-dependent format**     
+The DateBox formats its value according to the user's locale. This is the default behavior. You don't need to define the **displayFormat** property. Refer to the following help topic for more information: [Localize Dates, Numbers, and Currencies](/Documentation/Guide/Common/Localization/#Localize_Dates_Numbers_and_Currencies)
+
+- **Built-in predefined formats**    
+DevExtreme supports a number of [predefined formats](/Documentation/ApiReference/Common/Object_Structures/format/#type). Those are simple strings or shortcuts that define commonly used date formats so that you don't need to spell out a complex expression. Review the linked document for the list of values you can assign to **displayFormat**.  This demo uses the "shortdate" format. 
+
+- **LDML pattern**    
+Use an LDML pattern to construct a custom date-time format string. This demo sets **displayFormat** property to "EEEE, d of MMM, yyyy HH:mm" - a combination of day of the week, day number, month, year, hour, and minute. You can see the formatted value in the editor. Refer to the following help topic for information about supported format characters: [Custom Format String](/Documentation/Guide/Common/Value_Formatting/#Format_Widget_Values/Custom_Format_String)    
+
+- **Format with literal characters**    
+Specify the **displayFormat** property as a string that contains literal characters. This string can also contain LDML format characters. Wrap any characters that are not a part of the LDML pattern in quotation marks. Otherwise they may be interpreted as wildcards. In this example, if you remove quotation marks from the word "Year", the letter "a" will be interpreted as "AM/PM" placeholder.
+
+Note that the **DateBox** supports input masks. Use them to ensure that the input value format is consistent with **displayFormat**. To enable this behavior, set the [useMaskBehavior](/Documentation/ApiReference/UI_Widgets/dxDateBox/Configuration/#useMaskBehavior) value to *"true"*.
```

Comment:
```suggestion
You can also use input mask in the **DateBox**. Input masks ensure that the input value format is consistent with **displayFormat**. To do this, set the [useMaskBehavior](/Documentation/ApiReference/UI_Widgets/dxDateBox/Configuration/#useMaskBehavior) value to *"true"*.
```

Reply:
With all due respect, in my opinion, this is an unnecessary correction as it doesn't add any value, better inform users, improve grammar, etc. Moreover, using the word "input mask" in two consecutive sentences makes the text repetitive, which doesn't seem like an improvement over the previous version.  "You can also use input mask in....Input masks.... input....".

Would you care to elaborate on the reasoning behind these corrections?

Reply:
- There is no need to add 'Note this' or 'Note that'. If the DateBox allows you to use input masks, why not just state this?
-  'supports input masks' means what? You can use input masks in the DateBox? If that is the case, why not just say that?
- It is almost always better to say what **this/them/that** refers to
- I agree, an argument could be made to remove **input** from  'the input value format'

You can also use an input mask in the **DateBox**. Input masks ensure the entered value matches the **displayFormat**. To do this, set the [useMaskBehavior](/Documentation/ApiReference/UI_Widgets/dxDateBox/Configuration/#useMaskBehavior) value to *"true"*.

Reply:
Added this suggestion to the latest commit